sanic/config.py

```diff
@@ -140,6 +79,48 @@ def load_environment_vars(self, prefix=SANIC_PREFIX):
                             self[config_key] = v
 
 
+    def update_config(self, config: Union[bytes, str, dict, Any]):
+    """Update app.config.  
+    
+    Note only upper case settings are considered.  
+    
+    You can upload app config by providing path to py file holding settings.  
+    
+        # /some/py/file  
+        A = 1  
+        B = 2  
+    
+        config.update_config("${some}/py/file")  
+    
+    Yes you can put environment variable here, but they must be provided in format: ${some_env_var},  
+    and mark that $some_env_var is treated as plain string.  
+    
+    You can upload app config by providing dict holding settings.  
+    
+        d = {"A": 1, "B": 2}  
+        config.update_config(d)  
+    
+    You can upload app config by providing any object holding settings,  
+    but in such case config.__dict__ will be used as dict holding settings.  
+    
+        class C:  
+            A = 1  
+            B = 2  
+        config.update_config(c)"""
+    
+        if isinstance(config, (bytes, str)):
+            config = load_module_from_file_location("config", location=config)
+    
+        if not isinstance(config, dict):
+            config = config.__dict__
+    
+        config = dict(filter(lambda i: i[0].isupper(), config.items()))
+    
+        self.update(config)
+
+
+# Is in Sanic any better place where to keep this ???
```

Comment:
Keep what?

Reply:
Maybe we could find better place where to keep:  
strtobool()
load_module_from_file_location()
not inside config.py
because these function may by used elsewhere as well.

Reply:
Make a `utils.py` and place them there.

Reply:
OK.